contrib/apparmor/abstractions/electrum

```diff
@@ -0,0 +1,43 @@
+include <abstractions/base>
+include <abstractions/fonts>
+include <abstractions/user-tmp>
+include <abstractions/X>
+include <abstractions/wayland>
+include <abstractions/mesa>
+include <abstractions/dri-enumerate>
+include <abstractions/nameservice>
+include <abstractions/openssl>
+include <abstractions/vulkan>
+include <abstractions/python>
+include <abstractions/evince>
+include <abstractions/xdg-open> 
+include <abstractions/ubuntu-browsers>
+include <abstractions/snap_browsers>
```

Comment:
What about when not using Ubuntu, but instead e.g. Debian?
Are these defined on Debian? Or how do these includes work -- is not finding one of these a hard error or just a warning/silent ignore?

Reply:
Yes, they are configured on Debian (I tested on Debian 12 and also Kali). All the profiles are predefined, and if one is missing, AppArmor produces a hard error. 
We can also set them as `include if exists`  (last four) , as they are only needed by the help tab and don't cause any hard errors if blocked.
The strange thing I noticed is that `Kali` didn't come with the `evince profile`, but this was because `evince` wasn't installed, after installing the profile was installed.

Reply:
> We can also set them as `include if exists` (last four) , as they are only needed by the help tab and don't cause any hard errors if blocked.

I think we should that do then, yes.

> The strange thing I noticed is that Kali didn't come with the evince profile, but this was because evince wasn't installed, after installing the profile was installed.

It also does not make sense to tie us to a specific PDF reader program.
Specifically about evince, is it even installed if you choose a different DE than GNOME? E.g. if you install debian with KDE? Anyway, best not require it.